cli/command/utils.go

```diff
@@ -115,5 +117,23 @@ func PruneFilters(dockerCli Cli, pruneFilters filters.Args) filters.Args {
 		pruneFilters.Add(parts[0], parts[1])
 	}
 
+	// Server APIs older than 1.29 will just ignore label filters.
```

Comment:
@gdevillele yep, I did mis-understand the fix at first 😅.

We might (or should have) validates filters by version (for any api endpoint that supports them) in `docker`/`moby` 😓